src/Http/Http/src/Builder/ApplicationBuilder.cs

```diff
@@ -149,4 +165,12 @@ public RequestDelegate Build()
 
         return app;
     }
+
+    private static partial class Log
+    {
+        [LoggerMessage(1, LogLevel.Information,
+            "Request reached the end of the middleware pipeline {Protocol} {Method} {Scheme}://{Host}{PathBase}{Path}",
```

Comment:
This sounds too generic.
Suggestion:
`"Request reached the end of the middleware pipeline without being handled by application code. Returning status {StatusCode}. {Protocol} {Method} etc..."`